web/themes/joinup/prototype/sass/_drupal-hacks.sass

```diff
@@ -180,6 +180,8 @@ form .description
 
 // jQuery ui hacks
 .ui-dialog
+  width: auto !important
```

Comment:
![download_modal_issue](https://user-images.githubusercontent.com/1923476/30736752-562ad3c6-9f84-11e7-82cd-1fb187aeef81.png)

Not sure if that is a desired behavior? The wireframe is showing the modal differently.

Note: the other modal for leaving a collection is visualized ok.

Reply:
The problem has been created by adding additional lines of text. It should be fine now.